frontend/packages/app/src/clutch.config.js

```diff
@@ -75,5 +75,11 @@ module.exports = {
     details: {
       hideNav: true,
     },
+    config: {
+      hideNav: true,
+      componentProps: {
+        defaultRoute: "info",
```

Comment:
nit: can we make this `config`

Reply:
Yeah the issue is mostly from the workflow config, the path with `/:configType` cant be optionally set or it doesn't seem to know how to route to it and `/` doesn't satisfy it currently. So I had picked info just as a placeholder. But we can definitely remove it from the `clutch.config` file.